packages/weston.rb

```diff
@@ -19,29 +20,34 @@ class Weston < Package
     depends_on 'pango'
     depends_on 'dbus'
     depends_on 'libxxf86vm'
+    depends_on 'llvm' => :build
   end
 
-  binary_url ({
-      i686: 'https://dl.bintray.com/chromebrew/chromebrew/weston-4.0.0-chromeos-i686.tar.xz',
-    x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/weston-4.0.0-chromeos-x86_64.tar.xz',
-  })
-  binary_sha256 ({
-      i686: '477a86049694ee6021a2cf98aa8d85acec4684fdee07c6f2508babd470d594f2',
-    x86_64: '758ecfd17e47436b7c219676f5959379ed6a1ea7838abeeb82c54be456a47aa7',
-  })
 
   def self.build
-    system './autogen.sh'
-    system './configure',
-           "--prefix=#{CREW_PREFIX}",
-           "--libdir=#{CREW_LIB_PREFIX}",
-           '--disable-weston-launch',
-           '--enable-demo-clients-install'
-    system 'make'
+    ENV['CFLAGS'] = "-fuse-ld=lld"
+    ENV['CXXFLAGS'] = "-fuse-ld=lld"
+    system "meson #{CREW_MESON_OPTIONS} -Dshell-ivi=false -Dremoting=false -Dbackend-default=wayland -Dbackend-drm=false -Dpipewire=false -Dcolor-management-colord=false -Dcolor-management-lcms=false -Dbackend-rdp=false -Dlauncher-logind=false -Dweston-launch=false -Dsystemd=false -Dxwayland-path=#{CREW_PREFIX}/bin/Xwayland build"
+    system "meson configure build"
+    system 'ninja -C build'
   end
 
   def self.install
-    system "make DESTDIR=#{CREW_DEST_DIR} install"
+    system "DESTDIR=#{CREW_DEST_DIR} ninja install -C build"
+  end
+  
+  def self.postinstall
+    puts
+    puts "To complete the installation, execute the following:".lightblue
+    puts "mkdir -p ~/.config".lightblue
+    puts "echo '[core]' >> ~/.config/weston.ini".lightblue
+    puts "echo 'xwayland=true' >> ~/.config/weston.ini".lightblue
+    puts "echo '[terminal]' >> ~/.config/weston.ini".lightblue
+    # Cousine is the defaut ChromeOS monospace font.
+    puts "echo 'font=Cousine' >> ~/.config/weston.ini".lightblue
+    puts
+    puts "To run weston with xwayland try something like this:".lightred
+    puts "export WAYLAND_DISPLAY=wayland-1".lightred
+    puts "WAYLAND_DISPLAY=wayland-0 weston -Swayland-1 --xwayland".lightred
```

Comment:
Could we create an environment file that is sourced with the weston executable?  Maybe create a separate bash script to source the file?  That way, users wouldn't need to remember a bunch of configure options.

Reply:
I think weston is more for debugging use. Until it has acceleration and has rootless support, I can't see most people using it as default, since it is a reference compositor.

Reply:
Fair enough.